### PR TITLE
29-fix-android-inputaccessoryview-issue

### LINKED ISSUE #29: Fix android InputAccessoryView issue
Android app fails when screens using InputAccessoryView are open. 
Replace InputAccessoryView component with KeyboardAvoidingView

### PR DESCRIPTION
**Pull request by vitalikgriza**
Remove InputAccessoryView causing crash on android

### CODE REVIEW

**Comment by vitalikgriza** [comment]
Fixed crashes caused by InputAccessoryView on Android
Fixed icons on android
Fixed button component
Please note what there are not styling fixes

**Comment by vitalikgriza** [comment]
https://user-images.githubusercontent.com/42895774/183247325-fd88d910-ec68-467c-8538-df8152ddef85.mp4

**Review by hexadecible — APPROVED** [review]

File: src/screens/RegisterConsent/RegisterConsentScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+export type RegisterConsentScreenProps = NativeStackScreenProps<
+  RegisterStackParams,
+  'registerConsent'
+>;
+
+export const RegisterConsentScreen = (props: RegisterConsentScreenProps) => {
+  const headerHeight = useHeaderHeight();
+
+  const [selected1, setSelected1] = React.useState(false);
```
Comment (suleymancelikel) [inline src/screens/RegisterConsent/RegisterConsentScreen.tsx:26]: Maybe we can define more understandable names.  
exp: 
selected1 -> permissionConfirmation
selected2 -> policyApproval

Comment (vitalikgriza) [reply src/screens/RegisterConsent/RegisterConsentScreen.tsx:26]: Good point, but it was out of the scope of this task. For global refactoring we will have another task

Comment (vitalikgriza) [reply src/screens/RegisterConsent/RegisterConsentScreen.tsx:26]: Note: this file was moved, not created by me

**Pull request merged**